colabfold/batch.py

```diff
@@ -258,6 +265,21 @@ def predict_structure(
             )
             relaxed_pdb_path.write_text(relaxed_pdb_lines[r])
 
+        paes_path = result_dir.joinpath(
+            f"{prefix}_pae_{model_names[r]}_rank_{n + 1}.npy"
```

Comment:
I think there's a json format from alphafold for that data that's still on my todo list add, but for this the numpy format is good

Reply:
I think they are using the pickle library to export.

From deepmind alphafold:

```
`features.pkl` – A pickle file containing the input feature NumPy arrays used by the models to produce the structures.
```

I could easily implement it. However, there might be other useful model information to store in it.

Reply:
I replaced `np.save()` with `pickle`